readme.txt

```diff
@@ -22,6 +22,11 @@ Here is how to get started:
 
 Then open [a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) with the suggested changes.
 
+== How to use ==
```

Comment:
Maybe we could move it before the "Get Involved" section since usage instructions are more important than contributor instructions:

https://github.com/georgestephanis/two-factor/blob/dac65f43845cb9d0367d0771e7b0503363ab422d/readme.txt#L10-L25

Secondly, could it use the same title casing `How to Use` as the rest of the headings?

Reply:
Definitely. I simply wasn't sure what was more important to you.
And also sure about `How to Use`.

Reply:
About the logo - good call. However, plugins such as DUO, Google Authenticator – WordPress Two Factor Authentication (2FA) and Key two also use some kind of lock or keyhole as their icon.